File: server/server.Common/Entities/Car.cs
using server.Common.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace server.Common.Entities
{

    
    internal class Car
    {
        public int Id { get; set; }
        public string Model { get; set; }
        public Manufacturer Manufacturer { get; set; }
        public BodyType Body { get; set; }
        public int Mileage { get; set; }
        public int Seats { get; set; }
        public int Weight { get; set; }
        public string VIN { get; set; }
        public int HP { get; set; }
        public uint YearOfManufacture { get; set; }
        public FuelType FuelType { get; set; }

    }
}

File: client/src/models/entities/Car.ts
import { BodyType } from "../enums/BodyType";
import { FuelType } from "../enums/FuelType";
import { Manufacturer } from "./Manufacturer";

export interface Car {
  id: number;
  model: string;
  manufacturer: Manufacturer;
  body: BodyType;
  mileage: number;
  seats: number;
  weight: number;
  vin: string;
  hp: number;
  yearOfManufacture: number;
  price: number;
  fuelType: FuelType;
}
